### DISCUSSION TITLE
Apk trying to connect to my PC as server ....

### DISCUSSION THREAD

**Discussion by yeechutngeow** [quasarframework/quasar#14736] [CLI - Capacitor / Cordova modes]
But I am trying build a standalone mobile app that connects to firebase/firestore!
What did I missed?

Many thanks in advance.

**Comment by MilosPaunovic** [comment]
You haven't given us anything to work with. Share some code, how you're connecitng the app to the firebase, something...

**Comment by yeechutngeow** [comment]
Below is the landing page, which shows a google signin button, and the google SignIn logic.
Behind the scene is a Pinia store, with a firebase listener to monitor authentication change, and redirect the user to the main content page when a signin event is detected.
I ran the code in Android Studio, and created an apk for my handphone. In both cases the app shows a message that says:

"Using app server http://192.168.1.200:9500

which is pointing back to my development notebook. The message disappear shortly and I am left with a blank screen.
```
SignIn.vue
<template>
           ...
          <q-card-actions class="q-px-md">
            <q-btn class="g-signin-button full-width" @click="googleSignIn" label="Sign In With Google" />
          </q-card-actions>
</template>
<script setup>
import { getAuth, signInWithPopup, GoogleAuthProvider, setPersistence, indexedDBLocalPersistence } from 'firebase/auth';
import { useAuthStore } from 'stores/auth'

// to make sure that the authStore is instantiated
useAuthStore();

const provider = new GoogleAuthProvider();
const auth = getAuth();

function googleSignIn() {
  setPersistence(auth, indexedDBLocalPersistence)
    .then(() => {
      signInWithPopup(auth, provider);
    })
};
</script>

```

**Reply by yeechutngeow** [reply]
Another thing - the VS Code terminal shows about 29 of warning messages that look like the following:

Sourcemap for "D:/.../src-capacitor/node_modules/@capacitor/core/dist/esm/index.js" points to missing source files
Sourcemap for "D:/.../src-capacitor/node_modules/@capacitor/core/dist/esm/web/index.js" points to missing source files
Sourcemap for "D:/.../src-capacitor/node_modules/@capacitor/core/dist/esm/web-plugins.js" points to missing source files

**Comment by yeechutngeow** [comment]
My bad, the earlier problem was caused by a firebase dependencies I left in src-capacitor/package.json in the attempts to try to get Google Sigin to work.

Now I can get the login screen to show, but still, my development notebook IP is hardcoded into the capacitor.config.config as shown in the picture. Google SignIn now complains that "The current domain is not authorized for OAuth operations... Add your domain (192.168.1.200) to the OAuth redirect domains list in the Firebase console..."
 
![QuasarDevCapacitorAndroid](https://user-images.githubusercontent.com/2901215/198774075-3216aed3-8c5c-445b-8440-36f91f8be82f.png)

**Reply by MilosPaunovic** [reply]
This is not Quasar issue anymore, so I suggest you try to google the solution, starting with this:
https://stackoverflow.com/a/37351800
